### DISCUSSION TITLE
Eventos em duplas e clã vs clã

### DISCUSSION THREAD

**Discussion by Batatoones** [leogianfagna/IssueTracker#957] [Ideia]
### Servidor afetado

Todos

### Qual o propósito da ideia?

Ideia proposta é agradar o público e tentar agregar mais competividade entre os clãs

### Descrição da ideia

**Sugestão de eventos de duelos:**
Eventos de duelos sempre foi desejado pela comunidade de pvp então cabe a gente entregar o que a galera deseja. Minha ideia será voltada a 3 eventos pra começar em fase de testes! 

Primeiro evento: 2x2 com duplas formadas pelo gosto dos jogadores ou de maneira aleatória.
  - Duplas definidas pelos jogadores: pode criar algo monótono e gerar reclamações de que sempre será aqueles mesmo jogadores acabam ganhando o evento contudo com passar do tempo pode alterar visando entrada de novos jogadores na comunidade.
  - Duplas aleatórias: Pode criar conflito dentro da comunidade pois pessoas viciadas em pvp caso caíssem com um mobile, ficariam bravas pois mobile possui suas limitações. Lado bom da situação é que pode unir mais a comunidade e criar algo mais saudável contudo teria que conter sempre um adm nos eventos pra punir galera que falasse merda.

**Kit do evento:** 

> Sem McMmo;
> 1 armadura no corpo p4 diamante + af5 espada & machado;
> 32 caps;
> Todas pots speed 2 + força 2;
> Sem totem (visando estimular galera melhorar nos eventos e não batatar na cap além de deixar o evento algo mais rápido).

**Eventos de 2x2 e 3x3 em clãs!** 

Esse evento deveria acontecer em cada servidor pois poderia criar competividade em ambos servidores survival além de incentivar novos jogadores a participarem ou do vanillew ou do henesys visando ganhar esses eventos. Pode vir a faltar membros no henesys atualmente devido a falta de jogadores mas implementar mesmo assim. 
Motivo de criar esse tipo de evento que é algo que outros servidores possuem e também cria uma competividade pra galera competir por jogadores bons e de alguma maneira trazer eles pra clãs diferentes invés de todos permanecer em apenas um clã. Além disso, poderia ser algo que incentivaria nas ligas.

Os eventos teria 2x2 sem mcmmo e o 3x3 com mcmmo com kits diferentes cada um!

**2x2 kit:**

> Duas armaduras p4 diamante (uma no corpo + no inventário);
> 32 Caps
> 1 Totem
> Af 5 de machado & espada;
> Pots de força 2 e speed 2;
> Sem MCMmo.

**3x3 kit**

> Duas armaduras de p5 netherita (uma no corpo + no inventário);
> 32 Caps;
> 1 Totem;
> Af6 de machado & espada;
> Pots de força 2 e speed 2;
> Com McMmo (motivo? incentivar a galera farmar mcmmo e permanecer mais tempo no servidor pra competir).

### Aplicação

Tudo dito na descrição da ideia mas atualmente as arenas estão sendo feitas então acredito que isto pode ser usado pra dar um teaser e procurar criar um evento grande igual do herobrine foi atraindo membros antigos e novos.

Sugestão : Prints no discord e map arts dentro do jogo para incentivar fofocas e recompensar quem acertar com cash & vips!

**Comment by leogianfagna** [comment]
Já marcado em #966 e #967
